### ISSUE TITLE
star arts suggestion

### ISSUE THREAD

**Issue by androshalforc** [Azanor/thaumcraft-beta#1628]
tl:dr version
add a new research under star charts that allows you to collect star charts off of a master chart as needed/ supply them automatically to the research table if within range.

for the longest time ive had a love/hate relationship with the star charts system in TC.
i love the concept however the implementation seems to be lacking something. everything else in TC gets easier as you progress
-alchemy leads to the smeltery and thaumitorium.
-mining and fighting leads to auromancy.
-manual labour leads to golemancy.
however the star charts stay the same.

not that the star charts are difficult to obtain just time consuming over  and over again. particularly if you need that moon phase that just passed last night and now you need to wait for the whole cycle again.

what I propose is allowing the creation of something like placing 4 of the same star chart in a 2x2 grid of item frames and sprinkle with salis mundis. afterwords you can supply with a stack of paper and then clicking on the chart would add the appropriate chart to your inventory (or if possible count as being in your inventory if within range of the research table).
